docs/types/mapping-types.rst

```diff
@@ -52,6 +58,34 @@ each ``_KeyType``, recursively. For example with a mapping:
         }
     }
 
+The example below uses a mapping type inside another mapping type. The important
+point to remember are that the mapping inside the mapping is not a return variable,
+but the variable inside it is.
+
+::
+
+    pragma solidity >=0.4.0 <0.7.0;
+
+    contract MappingExample {
+        mapping(address => mapping(address => bool)) public validBalances;
```

Comment:
Doesn't erc20 use something like `allowances`?

Reply:
It does, but I'm not sure what you mean? This isn't really an ERC20 implementation, and the data structure here is different too.

Reply:
No, it's exactly the same - and I think it would be a good example: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol#L36